package.json

```
@@ -7,6 +7,7 @@
     "vip": "dist/bin/vip.js",
     "vip-app": "dist/bin/vip-app.js",
     "vip-app-list": "dist/bin/vip-app-list.js",
+    "vip-import-checker": "dist/bin/vip-app-check.js",
```

Oops, I think this is the wrong file path?

Also, minor- do we want the names to map to each other (i.e- either both `vip-import-check` or `vip-import-checker`)?
```suggestion
    "vip-import-checker": "dist/bin/vip-import-check.js",
```

---

Good catch.  I'd consider narrowing the name down to include `sql` in it as well (since there might be a media file import checker as well).

Perhaps something along the lines of `vip-sql-dump-checker`?

---

We should consider the name based out how users will be calling it. As it is now, they would call `vip import checker dbdump.sql`. Something about `checker` and `check` just sounds off. 

A better option might be `vip import validate dbdump.sql` or with more specificity (to @markowsiak's point) `vip import validate-sql dbdump.sql`.

---

changed to `vip import validate sql` so that we can have other sub-commands in the future 
https://github.com/Automattic/vip/pull/566/commits/0eb1cdc66523abb6b4359ff9bc7204e539bdc043